configure

```diff
@@ -6316,7 +6317,6 @@ enabled libsoxr           && require libsoxr soxr.h soxr_create -lsoxr
 enabled libssh            && require_pkg_config libssh libssh libssh/sftp.h sftp_init
 enabled libspeex          && require_pkg_config libspeex speex speex/speex.h speex_decoder_init
 enabled libsrt            && require_pkg_config libsrt "srt >= 1.3.0" srt/srt.h srt_socket
-enabled libtensorflow     && require libtensorflow tensorflow/c/c_api.h TF_Version -ltensorflow
```

Comment:
given that we still depend on the header files while building - would keeping it around like this work?

`enabled libtensorflow     && require_headers "tensorflow/c/c_api.h dlfcn.h"`

it's done similarly for frei0r filter

Reply:
Added the tensorflow/c/c_api.h requirement but did not add the dlfcn.h requirement, since that only exists on Linux and the requirement for libdl or LoadLibrary is taken care of above anyway.